modules/cold-extraction/ColdDataRetriever.py

```diff
@@ -587,6 +595,8 @@ def run_cold_extraction():
     ap.add_argument("--ThirdIndex", default=config['ThirdIndex'], type=int,
                     help="Set the CSV column index of third index query attribute. "
                          "Required only if the number of query attributes is 3")
+    ap.add_argument("--LongAccession", default=config['ThirdIndex'], type=bool,
```

Comment:
This should be, 
default=config['LongAccession']

Reply:
Modified the source code as per comments